### PR TITLE
KV - DriverAvailibilityTable, DriverAvailibilityUtils

### LINKED ISSUE #19: EPIC: Driver Availability - #4 DriverAvailabilityTable

### PR DESCRIPTION
**Pull request by Kavya75**
Added Driver Availability Table and its tests plus the driverAvailabilityUtils + test files. 
DriverAvailabilityTable storybook:

image

Closes #19

### CODE REVIEW

**Review by elaineeeho — APPROVED** [review]
LGTM

**Review by nguyenliliana — APPROVED** [review]
LGTM

File: frontend/src/main/components/Driver/DriverAvailabilityTable.js [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+        navigate(`/availability/edit/${cell.row.values.id}`)
+    }
+
+    // const deleteMutation = useBackendMutation(
+    //     cellToAxiosParamsDelete,
+    //     { onSuccess: onDeleteSuccess },
+    //     ["/api/driverAvailability/all"]
+    // );
+    
+    // const deleteCallback = async (cell) => { deleteMutation.mutate(cell); }
```
Comment (andrewpengucsb) [inline frontend/src/main/components/Driver/DriverAvailabilityTable.js:23]: Remove commented code

File: frontend/src/stories/components/Driver/DriverAvailibilityTable.stories.js [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+// RiderThreeSubjectsWithButtons.args = {
+//     ride: rideFixtures.threeRidesTable,
+//     currentUser: currentUserFixtures.userOnly
+// };
+
+// export const AdminThreeSubjectsWithButtons = Template.bind({});
+// AdminThreeSubjectsWithButtons.args = {
+//     ride: rideFixtures.threeRidesTable,
+//     currentUser: currentUserFixtures.adminUser
+// };
```
Comment (andrewpengucsb) [inline frontend/src/stories/components/Driver/DriverAvailibilityTable.stories.js:42]: Commented code

**Review by andrewpengucsb — CHANGES_REQUESTED** [review]

File: frontend/src/main/components/OurTable.js [hunk 3]
```diff
@@ -95,7 +95,7 @@ export function ButtonColumn(label, variant, callback, testid, displayText) {
         data-testid={`${testid}-cell-row-${cell.row.index}-col-${cell.column.id}-button`}
         className={cell.column.id} 
       >
-        {displayText}
+        {displayText || label}
```
Comment (andrewpengucsb) [inline frontend/src/main/components/OurTable.js:98]: I dont prefer changes to OurTable unless absolutely necessary. but I see that this is probably to fix the change introduced in a previous PR that maybe shouldn't have been allowed but I'll let it slide this time. So long as all the tables are still working as intended.

**Review by andrewpengucsb — APPROVED** [review]

**Review by nguyenliliana — APPROVED** [review]
LGTM

**Pull request merged**